docs/README.Windows.md

```diff
@@ -261,3 +263,30 @@ UWP builds generate `msix`, `appxsym` and `cer` files, located inside directorie
 
 
 **[back to top](#table-of-contents)** | **[back to section top](#6-build-kodi-manually)**
+
+## 7. Build binary add-ons
+You can find a complete list of available binary add-ons **[here](https://github.com/xbmc/repo-binary-addons)**.
+
+There are two approaches to building binary add-ons: _In-tree_ building that is automated, but dont allow to build from local sources (forks),
+and _out-of-tree_ building that allows building using local sources, which however will not be described here as it is not a simplified process,
+so refer to each add-on repository for more details.
```

Comment:
``
There are two approaches to build binary add-ons: _In-tree_ building that is automated, but doesn't support building from sources (forks, PRs, branches) other than the [Kodi binary addon repo](https://github.com/xbmc/repo-binary-addons),
and _out-of-tree_ building that allows building from any source. This can be unique for each add-on, therefore please refer to each add-on repository for more details.
``

I should possibly mention the context of in-tree and out-of-tree isnt exactly correct with how those terms are used most commonly, but i dont really think its worthwhile differentiating further here.

Reply:
yes no problem text replaced